src/core/components/hv-navigator/index.tsx

```diff
@@ -26,7 +30,7 @@ import { getFirstChildTag } from 'hyperview/src/services/dom/helpers';
 /**
  * Flag to show the navigator UIs
  */
-const SHOW_NAVIGATION_UI = false;
+const SHOW_NAVIGATION_UI = true;
```

Comment:
I think this override and commented use of `SHOW_NAVIGATION_UI` need to be reverted